source/extensions/tracers/xray/xray_configuration.h

```diff
@@ -1,22 +1,33 @@
 #pragma once
 #include <string>
 
-#include "absl/strings/string_view.h"
-
 namespace Envoy {
 namespace Extensions {
 namespace Tracers {
 namespace XRay {
 
+/**
+ * X-Ray configuration Model.
+ */
 struct XRayConfiguration {
-  XRayConfiguration(absl::string_view daemon_endpoint, absl::string_view segment_name,
-                    absl::string_view sampling_rules)
-      : daemon_endpoint_(daemon_endpoint), segment_name_(segment_name),
-        sampling_rules_(sampling_rules) {}
+  const std::string daemon_endpoint;
```

Comment:
nit: out style is to use an underscore suffix (e.g. `daemon_endpoint_`) This applies throughout the PR.